### PR TITLE
Removed the use of `method.call()` to try and get return values

### PR DESCRIPTION
**Pull request by dOrgJelli**

### CODE REVIEW

File: src/lib/integrations/daoStack/arc/createDao.ts [hunk 1]
```diff
@@ -74,19 +74,23 @@ export const createDao = async (
     cap
   )
 
-  const Avatar = await forgeOrg.call()
   let tx = await forgeOrg.send()
   console.log("Created new organization.")
   console.log(tx)
+  const Avatar = tx.events["NewOrg"].returnValues["_avatar"]
```
Comment (sogasg) [inline src/lib/integrations/daoStack/arc/createDao.ts]: This returns the same as `const Avatar = await forgeOrg.call()` does.

Comment (sogasg) [reply src/lib/integrations/daoStack/arc/createDao.ts]: I see that DAOstack has made the same change: https://github.com/daostack/migration/pull/116/files

Why is `const Avatar = await forgeOrg.call()` functioning for me??  😕 ❓

File: src/lib/integrations/daoStack/arc/createDao.ts [hunk 2]
```diff
@@ -225,16 +234,21 @@ export const createDao = async (
         })
       }
       console.log(deploymentInfo)
-      const setParams = schemeContract.methods.setParameters.apply(
-        null,
-        getSchemeCallableParamsArray(schemeConfig, deploymentInfo)
-      )
-      const schemeParametersKey = await setParams.call()
+      console.log(schemeConfig)
```
Comment (gh1dra) [inline src/lib/integrations/daoStack/arc/createDao.ts]: can we ditch this?

File: src/lib/integrations/daoStack/arc/createDao.ts [hunk 3]
```diff
@@ -225,16 +234,21 @@ export const createDao = async (
...
       const tx = await setParams.send()
       console.log(`${schemeConfig.typeName} parameters set.`)
       console.log(tx)
 
+      const getParamsHash = schemeContract.methods.setParameters.apply(
+        null,
+        args
+      )
+      const schemeParametersKey = await getParamsHash.call()
+      console.log(schemeParametersKey)
```
Comment (gh1dra) [inline src/lib/integrations/daoStack/arc/createDao.ts]: and this as well

File: src/lib/integrations/daoStack/arc/createDao.ts (hunk 1 above)
Comment (dOrgJelli) [reply src/lib/integrations/daoStack/arc/createDao.ts]: I'm not sure...

Comment (dOrgJelli) [reply src/lib/integrations/daoStack/arc/createDao.ts]: @sogasg I just tested this again in your branch and it's working... I'm not sure why though... it doesn't make sense to me that this code snippet works:
```
const Avatar = await forgeOrg.call()
let tx = await forgeOrg.send()
console.log(Avatar)
console.log(tx.events.NewOrg.returnValues._avatar)
```

What does `.call(...)` on a txMethod even mean? Why is it returning the correct result before the transaction has been sent? When doing `await forgeOrg.call()`, is it just running this on a single node "off chain" and returning back the result that should be?

Comment (dOrgJelli) [reply src/lib/integrations/daoStack/arc/createDao.ts]: Regardless, not sure why this wasn't working on my machine last night, I must be loosing it... I'm okay with abandoning this PR if we can have some sort of answer around why .call works.

File: src/lib/integrations/daoStack/arc/createDao.ts [hunk 4]
```diff
@@ -74,19 +74,23 @@ export const createDao = async (
...
   console.log(tx)
+  const Avatar = tx.events.NewOrg.returnValues._avatar
+  console.log(Avatar)
 
   const avatar = new web3.eth.Contract(
     require("@daostack/arc/build/contracts/Avatar.json").abi,
     Avatar,
     opts
   )
+  console.log(avatar)
```
Comment (gh1dra) [inline src/lib/integrations/daoStack/arc/createDao.ts]: pls remove unless you'd like to keep these for debugging other tix

**Pull request closed without merging**